src/logion/services/call.ts

```diff
@@ -1,5 +1,7 @@
 import { CallBase, AnyTuple, AnyJson } from '@polkadot/types/types';
 import { FunctionMetadataLatest } from "@polkadot/types/interfaces";
+import { UUID } from "@logion/node-api";
```

Comment:
Most types and functions defined below are now available from logion/node-api (see https://github.com/logion-network/logion-api/blob/main/packages/node-api/src/Extrinsics.ts).